src/main/java/Deadline.java

```diff
@@ -0,0 +1,27 @@
+public class Deadline extends Task{
+
+    protected String by;
+
+    public Deadline(String description){
+        super(description);
+        //this.by = by;
+    }
+
+    public void setBy(String description){
+        String[] arrOfStr = description.split("/", 2);
+        by = arrOfStr[1];
+        this.description = arrOfStr[0];
+        arrOfStr = by.split(" ", 2);
+        by = arrOfStr[0] + ": " + arrOfStr[1];
+    }
+
+    public String getBy(){
+        return by;
+    }
+
+    public String toString(){
+
```

Comment:
Can consider to delete this extra unnecessary blank line so that code is neater and of better quality.
Similarly for the rest of the code in the other classes.